plugins/registry.js

```diff
@@ -79,6 +79,29 @@ export default class PluginRegistry {
         return dispatchPluginComponentAction('LeftSidebarHeader', this.id, component);
     }
 
+    // Register an App in the list of apps.
+    // Accepts a React component. Returns a unique identifier.
+    registerTeamAppComponent(component, show = () => true) {
```

Comment:
Not so much beta, I have been using it for months in production.
This registers a menu entry at the top of the channel sidebar where you can add quick links to URLs and/or plugins

![FireShot Capture 225 - Town Square - Titans Mattermost - localhost](https://user-images.githubusercontent.com/1237191/73296945-f245fa80-420a-11ea-9bc1-90004a1542d6.png)

you can [see an animated gif here](https://github.com/SpikeTings/tings-centerapptest)

Reply:
@aaronrothschild, can you provide some thoughts on how this meshes with your vision of spaces in the bot channels? I'm wondering if we shouldn't instead enforce that this experience occurs in the context of a bot, and provide a way to override the centre channel for that, instead of a "hard-coded" `APPLICATIONS` section that will likely collide with the upcoming sidebar reorganization by the apps team. cc @deanwhillier